docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
@gian took a look at same, expression language used there is json-path which is specific to json format not necessarily suited for avro records.
unlike json, avro world does not have an standard expression language for extracting fields and we decided to borrow same from http://commons.apache.org/proper/commons-beanutils/javadocs/v1.9.2/apidocs/org/apache/commons/beanutils/package-summary.html#standard.basic that looked general enough to fulfill the need here.

similarity i see is that we can rename "schemaMappings" to "flattenSpec" . 

also, there is no "compatibility" issue... this module defines brand new parsers which do not have much to do with json parsers. can you please elaborate on the compatibility issue?